aquacomputer_d5next.c

```diff
@@ -476,12 +493,25 @@ static int aqc_percent_to_pwm(u16 val)
 /* Converts to centi-percent */
 static int aqc_pwm_to_percent(long val)
 {
-	if (val < 0 || val > 255)
-		return -EINVAL;
-
 	return DIV_ROUND_CLOSEST(val * 100 * 100, 255);
 }
 
+/* Converts raw value for Aquastream XT pump speed to rpm */
+static int aquastream_convert_pump_rpm(s16 val)
+{
+	if (val > 0)
+		return DIV_ROUND_CLOSEST(45000000, val);
+	return 0;
+}
+
+/* Converts raw value for Aquastream XT fan speed to rpm */
+static int aquastream_convert_fan_rpm(s16 val)
+{
+	if (val > 300)
```

Comment:
Why exactly 300? What does it signify? Magic constants should be macros. Same with 5646000 and 45000000.